### DISCUSSION TITLE
Kanban Board

### DISCUSSION THREAD

**Discussion by Pemo11235** [kudrykv/latex-yearly-planner#28] [Ideas]
Hi ! ✋🏻  

Is there a way to implement a Kanban Board in weekly pages ? 
It would be so cool to work with !

Thanks for your amazing job  🚀

**Comment by kudrykv** [comment]
Hi @Pemo11235!

Can you please provide an example of what you envision?

**Reply by Pemo11235** [reply]
I saw a planner that has a kanban board for each week. 
Each day has a link to board , and every week day is linked in kanban board's right corner.

This very suits my workflow and task management.

Thanks for your time 🙂

**Reply by kudrykv** [reply]
It's a neat idea!

Currently, I'm working on a refactor, so I'm occupied from that perspective. Anyway, I'll put a pin on this idea! This sounds like a nice thing to have neighboring weekly pages.

**Reply by Pemo11235** [reply]
Cool ! It's a shame that I'm not confident with Go to give my contribution.

Looking forward for you refactor!

Have a nice day mate 🙂

**Reply by tschnibo** [reply]
@Pemo11235  Hey :) just to understand this correctly - if you want, can you maybe provide a sample view.? How is this different from the normal "weekly overview"? Just "extra" and a different design approach? Or are there other "links"? Nice day to you all! :)

**Reply by Pemo11235** [reply]
> @Pemo11235 Hey :) just to understand this correctly - if you want, can you maybe provide a sample view.? How is this different from the normal "weekly overview"? Just "extra" and a different design approach? Or are there other "links"? Nice day to you all! :)

@tschnibo Is there this planner  https://hyperpaper.me/ that implement Kanban Board as weekly page (there is a link on the top-right corner for every day of the week) :  
![Immagine 2022-07-10 114305](https://user-images.githubusercontent.com/54067721/178139637-8090a497-7de3-4709-b1b0-e77d5422f75b.png)

Then, every day has a link to the kanban board of the week:  
![dd](https://user-images.githubusercontent.com/54067721/178139676-662d3b09-09c9-4003-b08c-66fe182151ed.png)

IMHO, it would works good if  there is a Kaban Board for each week. 
I would put a link  Kanban<->Week-View only, because a link Kanbam<->Each-Day would me a little messy maybe.

**Reply by tschnibo** [reply]
@Pemo11235 Thank you a lot!
I see how this is meant to be :)

@kudrykv besides the whole SN / Remarkable layout which I don't have readily at hand. The Kanban pieces I could solve with my tcolorbox approach, similar to:
![grafik](https://user-images.githubusercontent.com/16188592/178142496-041e54e0-899f-4617-b7fe-85ca0ff01d00.png)

So maybe, if you would like to refactor together, I could maybe try to implement my "ways" for your standard formats too. Because this is quite some work, I would do this only in case you see a benefit in this. Maybe you have plenty ideas already :1st_place_medal:
